drivers/hmis/app/graphql/types/hmis_schema/project.rb

```diff
@@ -73,5 +75,9 @@ def inventories(**args)
     def units(**args)
       resolve_units(**args)
     end
+
+    def households(**args)
+      resolve_households(**args)
```

Comment:
why not use the loader here?

Reply:
So I don't think the loader does pagination correctly. I believe it pulls ALL households for a project, not a paginated set, so you'd still have to do another query to get paginated households. Figuring out how to make the loaders work with pagination is on my list of tricky problems I want to solve, but it's not solved yet.

Reply:
Oh okay, interesting. @ttoomey was just asking me the other day why we don't use the data loaders practically anywhere, and I didn't have an answer for that. So just heads up Theron if you're poking at that.